value_test.go

```diff
@@ -724,8 +725,12 @@ func TestReadOnlyOpenWithPartialAppendToWAL(t *testing.T) {
 	opts.ReadOnly = true
 	// Badger should fail a read-only open with values to replay
 	_, err = Open(opts)
-	require.Error(t, err)
-	require.Regexp(t, "Log truncate required", err.Error())
+	if runtime.GOOS == "windows" {
+		require.Equal(t, err, ErrWindowsNotSupported)
```

Comment:
could you just return after this line? and not have the else block

Reply:
done